clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/security/GoogleCredentialsInitializer.groovy

```diff
@@ -35,6 +37,15 @@ import org.springframework.context.annotation.Scope
 @Configuration
 class GoogleCredentialsInitializer implements CredentialsInitializerSynchronizable {
 
+
+  @Autowired
+  GoogleExecutor google_executor  // Not used, just here to force initialization ordering
```

Comment:
Camelcase.

Reply:
Also, can add an underscore to the beginning of this field name to indicate it's not used: `_googleExecutor`.